packages/components/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### Internal
 
 -   `Popover`, `ColorPicker`: Obviate pointer event trap #59449 ([#59449](https://github.com/WordPress/gutenberg/pull/59449)).
+-   `InputBase`: Simplify management of focus styles. Affects all components based on `InputControl` (e.g. `SearchControl`, `UnitControl`), as well as `SelectControl` and `CustomSelectControl` ([#60226](https://github.com/WordPress/gutenberg/pull/60226)).
```

Comment:
Should we list them all? May be easier if someone is looking for the changelog in the case of a regression.

Reply:
It's impractical to list all of them because so many compound components include some form of these inputs, but I added a few more to cover all the relatively primitive ones. I think that should be sufficient because those are the ones more likely to involve consumer overrides.